### PR TITLE
Hwkmetrics 309

### PR DESCRIPTION
**Pull request by auszone**

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

File: core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/MetricsServiceITest.java [hunk 1]
```diff
@@ -288,55 +288,50 @@ public void createAndFindMetricsWithTags() throws Exception {
...
-
-        metricsService.createMetric(mA).toBlocking().lastOrDefault(null);
-        metricsService.createMetric(mB).toBlocking().lastOrDefault(null);
-        metricsService.createMetric(mC).toBlocking().lastOrDefault(null);
-        metricsService.createMetric(mD).toBlocking().lastOrDefault(null);
-        metricsService.createMetric(mE).toBlocking().lastOrDefault(null);
-        metricsService.createMetric(mF).toBlocking().lastOrDefault(null);
-        metricsService.createMetric(mG).toBlocking().lastOrDefault(null);
+        for (Metric m : metricsToAdd) {
+            metricsService.createMetric(m).toBlocking().lastOrDefault(null);
```
Comment (burmanm) [inline core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/MetricsServiceITest.java]: This should not use a for-clause, instead an observable. Like: Observable.from(metricsToAdd).subscribe(m -> metricsService.createMetric(m)).. etc add blocking mechanism at the end.

Comment (auszone) [inline core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/MetricsServiceITest.java]: right

**Comment by stefannegrea** [comment]
Thanks @auszone! Good improvements.

**Comment by tsegismont** [comment]
This is nice but it does not solve HWMETRICS-309. The JIRA is about "rest-tests" module, not the core service. I will reopen the JIRA but please in the future don't use unrelated names for pull requests as the Github/JIRA link auto-closes issues.

**Pull request merged**